dev_apps/first-actor/intro.md

```diff
@@ -1,4 +1,12 @@
-## wash101
+# Your First wasmcloud Actor!
 
-## Your first actor
-In this tutorial, you will deploy your first actor using the wash REPL
+Congrats on making it this far!  Now it is time to take the take what we have learn from the basics courses and build our first actor.  How about a calculator?!
+
+In this tutorial, we have provided you with 3 implementations of the same calculator app.  They are written in Rust, Golang, and AssemblyScript.  You can choose your own adventure, or you can do them all!  One of the tenants of wasmcloud is that it allows you to build on the polyglot power of Webassembly.  
```

Comment:
written in Rust, Go, and AssemblyScript

by convention, we don't use `golang` unless in metadata/SEO and search requests. The name of the language is Go.

Reply:
s/tenants/tenets
s/wasmcloud/wasmCloud

Reply:
> Webassembly
WebAssembly